shell/platform/darwin/ios/framework/Source/FlutterEngineGroup.mm

```diff
@@ -79,7 +64,7 @@ - (FlutterEngine*)makeEngineWithOptions:(nullable FlutterEngineGroupOptions*)opt
                              initialRoute:initialRoute
                            entrypointArgs:entrypointArgs];
   }
-  [_engines addObject:[NSValue valueWithPointer:engine]];
+  [self.engines addObject:[NSValue valueWithPointer:(__bridge void*)engine]];
```

Comment:
curious what error message did you get without this cast?

Reply:
The error is:
```
error: cannot initialize a parameter of type 'const void * _Nullable' with an rvalue of type 'id _Nullable'
```

Either way, this needs to be bridged as a raw C pointer though to preserve existing semantics.